### DISCUSSION TITLE
Round IDs for submission files

### DISCUSSION THREAD

**Discussion by annakrystalli** [hubverse-org/hubValidations#6] [Q&A]
In hubs that have round specific configurations (i.e. where `round_id_from_variable: false`), how do teams signal which round they are submitting a given file too? @LucieContamin ?

**Comment by LucieContamin** [comment]
So currently all our rounds have the `round_id_from_variable: true` with the `origin_date` column, however the round_id is also in the file name. So, we should technically be able to do that (have `round_id_from_variable: false`) and only use the filename to check the round they are submitting.

**Comment by annakrystalli** [comment]
Aha! OK interesting. I've just come to realise that all our current example hubs then use `round_id_from_variable: true` so the next example we develop should be one that uses round IDs to determine rounds.
